aquacomputer_d5next.c

```diff
@@ -654,6 +691,451 @@ static void aqc_debugfs_init(struct aqc_data *priv)
+static SENSOR_DEVICE_ATTR_2_RW(pwm4_auto_point7_pwm, aqc_fan_curve_pwm, 3, 6);
+static SENSOR_DEVICE_ATTR_2_RW(pwm4_auto_point8_pwm, aqc_fan_curve_pwm, 3, 7);
+static SENSOR_DEVICE_ATTR_2_RW(pwm4_auto_point9_pwm, aqc_fan_curve_pwm, 3, 8);
+static SENSOR_DEVICE_ATTR_2_RW(pwm4_auto_point10_pwm, aqc_fan_curve_pwm, 3, 9);
+static SENSOR_DEVICE_ATTR_2_RW(pwm4_auto_point11_pwm, aqc_fan_curve_pwm, 3, 10);
+static SENSOR_DEVICE_ATTR_2_RW(pwm4_auto_point12_pwm, aqc_fan_curve_pwm, 3, 11);
+static SENSOR_DEVICE_ATTR_2_RW(pwm4_auto_point13_pwm, aqc_fan_curve_pwm, 3, 12);
+static SENSOR_DEVICE_ATTR_2_RW(pwm4_auto_point14_pwm, aqc_fan_curve_pwm, 3, 13);
+static SENSOR_DEVICE_ATTR_2_RW(pwm4_auto_point15_pwm, aqc_fan_curve_pwm, 3, 14);
+static SENSOR_DEVICE_ATTR_2_RW(pwm4_auto_point16_pwm, aqc_fan_curve_pwm, 3, 15);
+
+static SENSOR_DEVICE_ATTR_2_RW(pwm4_auto_point1_temp, aqc_fan_curve_temp, 3, 0);
+static SENSOR_DEVICE_ATTR_2_RW(pwm4_auto_point2_temp, aqc_fan_curve_temp, 3, 1);
+static SENSOR_DEVICE_ATTR_2_RW(pwm4_auto_point3_temp, aqc_fan_curve_temp, 3, 2);
+static SENSOR_DEVICE_ATTR_2_RW(pwm4_auto_point4_temp, aqc_fan_curve_temp, 3, 3);
+static SENSOR_DEVICE_ATTR_2_RW(pwm4_auto_point5_temp, aqc_fan_curve_temp, 3, 4);
+static SENSOR_DEVICE_ATTR_2_RW(pwm4_auto_point6_temp, aqc_fan_curve_temp, 3, 5);
+static SENSOR_DEVICE_ATTR_2_RW(pwm4_auto_point7_temp, aqc_fan_curve_temp, 3, 6);
+static SENSOR_DEVICE_ATTR_2_RW(pwm4_auto_point8_temp, aqc_fan_curve_temp, 3, 7);
+static SENSOR_DEVICE_ATTR_2_RW(pwm4_auto_point9_temp, aqc_fan_curve_temp, 3, 8);
+static SENSOR_DEVICE_ATTR_2_RW(pwm4_auto_point10_temp, aqc_fan_curve_temp, 3, 9);
+static SENSOR_DEVICE_ATTR_2_RW(pwm4_auto_point11_temp, aqc_fan_curve_temp, 3, 10);
+static SENSOR_DEVICE_ATTR_2_RW(pwm4_auto_point12_temp, aqc_fan_curve_temp, 3, 11);
+static SENSOR_DEVICE_ATTR_2_RW(pwm4_auto_point13_temp, aqc_fan_curve_temp, 3, 12);
+static SENSOR_DEVICE_ATTR_2_RW(pwm4_auto_point14_temp, aqc_fan_curve_temp, 3, 13);
+static SENSOR_DEVICE_ATTR_2_RW(pwm4_auto_point15_temp, aqc_fan_curve_temp, 3, 14);
+static SENSOR_DEVICE_ATTR_2_RW(pwm4_auto_point16_temp, aqc_fan_curve_temp, 3, 15);
+
+static SENSOR_DEVICE_ATTR_2_RW(pwm5_auto_point1_pwm, aqc_fan_curve_pwm, 4, 0);
+static SENSOR_DEVICE_ATTR_2_RW(pwm5_auto_point2_pwm, aqc_fan_curve_pwm, 4, 1);
+static SENSOR_DEVICE_ATTR_2_RW(pwm5_auto_point3_pwm, aqc_fan_curve_pwm, 4, 2);
+static SENSOR_DEVICE_ATTR_2_RW(pwm5_auto_point4_pwm, aqc_fan_curve_pwm, 4, 3);
+static SENSOR_DEVICE_ATTR_2_RW(pwm5_auto_point5_pwm, aqc_fan_curve_pwm, 4, 4);
+static SENSOR_DEVICE_ATTR_2_RW(pwm5_auto_point6_pwm, aqc_fan_curve_pwm, 4, 5);
+static SENSOR_DEVICE_ATTR_2_RW(pwm5_auto_point7_pwm, aqc_fan_curve_pwm, 4, 6);
+static SENSOR_DEVICE_ATTR_2_RW(pwm5_auto_point8_pwm, aqc_fan_curve_pwm, 4, 7);
+static SENSOR_DEVICE_ATTR_2_RW(pwm5_auto_point9_pwm, aqc_fan_curve_pwm, 4, 8);
+static SENSOR_DEVICE_ATTR_2_RW(pwm5_auto_point10_pwm, aqc_fan_curve_pwm, 4, 10);
```

Comment:
you missed this one.
id should be 9

Reply:
thank you very much, I should sleep more :heart: